src/conn/conn_dhandle.c

```diff
@@ -331,6 +331,11 @@ __wt_conn_btree_open(
 	F_SET(btree, LF_ISSET(WT_BTREE_SPECIAL_FLAGS));
 
 	WT_ERR(__wt_btree_open(session, cfg));
+	if (F_ISSET(dhandle, WT_DHANDLE_EXCLUSIVE) &&
```

Comment:
@ddanderson I don't really understand the condition here. Could you add a comment to explain why it's necessary?